storage/txcache/txListBySenderMap.go

```diff
@@ -1,26 +1,28 @@
 package txcache
 
 import (
-	"sort"
-
 	"github.com/ElrondNetwork/elrond-go/core"
 	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/storage/txcache/maps"
 )
 
+const numberOfScoreChunks = uint32(100)
+
 // txListBySenderMap is a map-like structure for holding and accessing transactions by sender
 type txListBySenderMap struct {
-	backingMap      *ConcurrentMap
-	counter         core.AtomicCounter
-	nextOrderNumber core.AtomicCounter
+	backingMap  *maps.BucketSortedMap
+	cacheConfig *CacheConfig
```

Comment:
why not having `cacheConfig CacheConfig` (without a pointer)

Reply:
Changed. Note that `txListForSender` still has the pointer (it has to be lightweight).